stainless_extraction/src/lib.rs

```diff
@@ -488,7 +488,7 @@ impl<'a, 'l, 'tcx> BodyExtractor<'a, 'l, 'tcx> {
         // substitute the type parameters of the field with the actual types from the key
         tps
           .iter()
-          .map(|t| type_substs.get(t).map(|&t| t))
+          .map(|t| type_substs.get(t).copied())
```

Comment:
Just discovered the equivalence of those two today. 😅

Reply:
Yeah, `.map(|&t| t)`, `.map(|t| *t)` and `.copied()` are all equivalent here :)